components/SingleReview.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { reviewById } from "../utils/data";
+
+const SingleReview = () => {
+    const [review, setReview] = useState({});
+   
+    const { review_id } = useParams();
+  useEffect(() => {
+    reviewById(review_id)
+      .then(({ data: { review } }) => {
+              setReview(review) })
+ 
```

Comment:
Nice object destructuring here, this can be tricky with nested objects. Good work